ssl/ssl_lib.c

```diff
@@ -1355,6 +1361,11 @@ int SSL_set_wfd(SSL *s, int fd)
         }
         BIO_set_fd(bio, fd, BIO_NOCLOSE);
         SSL_set0_wbio(s, bio);
+#ifndef OPENSSL_NO_KTLS
+        if (ktls_enable(fd)) {
+            printf("setsockopt failed %d\n", errno);
```

Comment:
same error-code/printf issue.

Reply:
See comment in crypto/bio/bss_sock.c. I'll fix it throughout the patchset according to what we agree about there.